### DISCUSSION TITLE
Kairos use UEFI or boot loader legacy

### DISCUSSION THREAD

**Discussion by massimogallina** [kairos-io/kairos#1906] [Q&A] [answered]
I would need an information about Kairos: do the ISOs generated with Kairos use UEFI or boot loader legacy?

Thansk,
M

**Comment by Itxaka** [comment]
Hey @massimogallina 

Generated isos are hybrid, they support booting both from BIOS and EFI systems.

**Comment by massimogallina** [comment]
Thanks @Itxaka. And how can I understand if it has been installed in UEFI mode? I am using the debian flavor.
It is sufficent to check the presence of the directory ` /sys/firmware/efi`?

**Reply by Itxaka** [reply, marked as answer]
yep, that is what kairos-agent uses to determine what mode to install from, if `/sys/firmware/efi` exists it will install/boot in EFI mode, otherwise it will use BIOS mode.
